src/adhocracy_core/docs/deletion.rst

```diff
@@ -1,123 +1,123 @@
 .. _api-deletion:
 
-Deleting and Hiding Resources
-=============================
+Deleting Resources
+==================
+
+Anyone with the *delete_resource* permission can *delete* it by using
+the HTTP DELETE verb.  Deleted resources can not be recovered.
+
+Deleting an existing resource is only possible for updatable
+resources, i.e. *not* for Versions, as this would mess up the version
+DAG.
+
+The effect of deleting is as follows:
+
+* All child/descendant resources (whose resource path includes the path
+  of the deleted ancestor as prefix) are also deleted.
+* Deleted resources are no longer listed in parent pools or search
+  queries.
+* If the frontend attempts to retrieve a *deleted* resource via
+  GET, the backend responds with HTTP status code *404 Not Found*, just
+  as if the resource had never existed.
+* *Deleted* resources may still be referenced from other
+  resources. If the frontend follows such references it must therefore
```

Comment:
This does not work with real deletion. Tthe frontend might have an outdated reference to an deleted resource, but the backend itself will delete all references/backreferences to deleted resources.